exonum/benches/criterion/block.rs

```diff
@@ -106,7 +106,12 @@ fn create_blockchain(db: impl Database, services: Vec<Box<dyn Service>>) -> Bloc
 }
 
 fn execute_block(blockchain: &Blockchain, height: u64, txs: &[Hash]) -> (Hash, Patch) {
-    blockchain.create_patch(ValidatorId::zero(), Height(height), txs)
+    blockchain.create_patch(
+        ValidatorId::zero(),
+        Height(height),
+        txs,
```

Comment:
So I think that we have to unify this to arguments by the single trait `TransactionsSource` with the good interface in future.